src/libs/NumberCovert.js

```diff
@@ -0,0 +1,57 @@
+import lodashGet from 'lodash/get';
+import loadashTrim from 'lodash/trim';
+import loadashTrimStart from 'lodash/trimStart';
+import loadashIncludes from 'lodash/includes';
+import loadashStartsWith from 'lodash/startsWith';
+import loadashPadStart from 'lodash/padStart';
+import translations from '../languages/translations';
+
```

Comment:
Can you name this file `LocalePhoneNumber.js` instead of `NumberCovert.js` please?

Reply:
Done.